### ISSUE TITLE
[Feature] Easier example project access

### ISSUE THREAD

**Issue by SolarEarth37** [ECDAR-AAU-SW-P5/Ecdar-GUI-Web#178]
### Description
The example projects are not easily accessible, after having started a new project. It would be beneficial to create a way for the user to either access them from the top bar, or an easy way back to the home page where they are currently located.

### Reason For Request
During a usability test, it was found that there was no way of opening an example project, after a blank project had been opened.

**Comment by atjn** [comment]
Agreed, that is also why the top bar has a button for example projects. The button does not yet work because we need to support some sort of submenu to choose the example project.
